### ISSUE TITLE
Optimize UUID conversion to and from String

### ISSUE THREAD

**Issue by fabiolimace** [f4b6a3/uuid-creator#27]
The methods to be optimized are: 
* `UuidConverter.fromString()`;
* `UuidCroverter.toString()`;

**Comment by fabiolimace** [comment]
Now the method `UuidConverter#fromString(String)` is **5x** faster than `UUID#fromString(String)` in JDK 8 and 11.

Now the method `UuidConverter#toString(String)` is **7x** faster than
UUID#toString() in JDK 8.

**Benchmark in JDK 8:**

```
Benchmark                           Mode  Cnt      Score     Error   Units
Throughput.UUID_fromString         thrpt    5   1917,663 ±  58,287  ops/ms
Throughput.UUID_toString           thrpt    5   2897,979 ±  99,724  ops/ms
Throughput.UuidCreator_fromString  thrpt    5  10230,760 ± 106,844  ops/ms
Throughput.UuidCreator_toString    thrpt    5  20800,729 ± 220,021  ops/ms
```

**Benchmark in JDK 11**

```
Benchmark                           Mode  Cnt      Score     Error   Units
Throughput.UUID_fromString         thrpt    5   7900,104 ±  78,608  ops/ms
Throughput.UUID_toString           thrpt    5  31687,104 ± 204,545  ops/ms
Throughput.UuidCreator_fromString  thrpt    5   9635,343 ±  62,319  ops/ms
Throughput.UuidCreator_toString    thrpt    5  20652,287 ± 336,251  ops/ms
```

**Comment by fabiolimace** [comment]
Released version 3.1.2.
